packages/iTwinUI-react/src/core/ButtonGroup/ButtonGroup.test.tsx

```diff
@@ -79,6 +79,47 @@ it.each(['start', 'end'] as const)(
   },
 );
 
+it.each(['start', 'end'] as const)(
+  'should handle overflow when available space is smaller than one element (%s)',
+  (overflowPlacement) => {
+    const scrollWidthSpy = jest
+      .spyOn(HTMLElement.prototype, 'scrollWidth', 'get')
+      .mockReturnValue(200);
+    const offsetWidthSpy = jest
+      .spyOn(HTMLElement.prototype, 'offsetWidth', 'get')
+      .mockReturnValue(50);
+
+    const OverflowGroup = () => {
+      const buttons = [...Array(3)].map((_, index) => (
```

Comment:
Maybe we can directly use Array(3) instead of using the spread operator? The tests passed even after making this change.

Reply:
So I copied this from the overflow test that was already there and didn't think about why it was that way. I did some research and testing though.

This stackoverflow question has some good explanations for the difference: https://stackoverflow.com/questions/66077917/creating-array-with-spread-operator
I particularly like the examples given in the top comment.

I also put some `console.logs` in the code to see the difference between using the spread operator vs not using it:
![image](https://user-images.githubusercontent.com/37936169/190433690-c4c0a07f-0a28-4843-a0d5-a22ae21f8803.png)
![image](https://user-images.githubusercontent.com/37936169/190433041-2eafbfed-4a43-49ba-94ff-950729bb860e.png)
The results show that the spread operator is necessary for `.map` to work properly.

Additionally, the tests passed because only one button was created without the spread operator, and my test was only looking for one button.

Reply:
Ohh, I meant like `Array(3).map(...` and not `[Array(3)].map(...`. I think that might work because at least from what I know, `[...Array(3)]` just creates a duplicate (new object in memory with the same contents) of `Array(3)`. 

Here is an example
![image](https://user-images.githubusercontent.com/45748283/190440110-74e6005f-7ee3-4c77-88cc-fce609196bfc.png)

Reply:
Hm, I tested it with just Array(3) and console.log shows a blank space instead of an array of objects. Also using `Array(3)` instead of `[...Array(3)]` in the existing test causes failures. So this doesn't work.

Reply:
Ohh, sorry, I don't think I properly saw the StackOverflow link you sent. After trying it out myself, l realize that `[...Array(3)]` is correct. `Array(3)` is not iterable, so we need `...[Array(3)]` since we need to iterate through it. I learned something new! :smile:

Reply:
Me too!